testapps/test_pillowtop/tests/test_settings.py

```diff
@@ -1,12 +1,13 @@
 import os
 from django.conf import settings
-from django.test import TestCase
+from django.test import TestCase, override_settings
 import json
 from corehq.util.test_utils import TestFileMixin
 from pillowtop.listener import AliasedElasticPillow, BasicPillow
 from pillowtop.utils import get_all_pillow_configs
 
 
+@override_settings(DEBUG=True)
```

Comment:
Yea, just wondering if there should be an option in soft assert to also
fail in unit tests.

On 10 November 2015 at 20:56, Cory Zue <notifications@github.com> wrote:

> In testapps/test_pillowtop/tests/test_settings.py
> <https://github.com/dimagi/commcare-hq/pull/9046#discussion_r44448037>:
>
> >  import json
> >  from corehq.util.test_utils import TestFileMixin
> >  from pillowtop.listener import AliasedElasticPillow, BasicPillow
> >  from pillowtop.utils import get_all_pillow_configs
> >
> >
> > +@override_settings(DEBUG=True)
>
> yeah, i ran into this
> <https://docs.djangoproject.com/en/1.4/topics/testing/#other-test-conditions>
> trying to make a test fail earlier
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/9046/files#r44448037>.
>



-- 
Simon Kelly
Senior Engineer | Dimagi South Africa